spec/models/mentor/student_spec.rb

```diff
@@ -7,7 +7,7 @@
     it { is_expected.to respond_to :coding_level     }
     it { is_expected.to respond_to :code_samples     }
     it { is_expected.to respond_to :learning_history }
-    it { is_expected.to respond_to :code_background  }
+    it { is_expected.to respond_to :language_learning_period }
```

Comment:
It's not a must, but it kinda hurts my eye to not see the `}`'s aligned 😉 
There's "aligning" or "not aligning", but the compromise, I don't know...